app/controllers/component-tree.js

```diff
@@ -194,10 +194,18 @@ class RenderItem {
     this.renderNode = renderNode;
   }
 
+  get args() {
+    return this.renderNode.args;
+  }
+
   get id() {
     return this.renderNode.id;
   }
 
+  get isCurlyInvocation() {
+    return this.renderNode && this.renderNode.args && this.renderNode.args.positional;
+  }
+
```

Comment:
Minor but can you move both `get args` and `get isCurlyInvocation` to _after_ `get name`? (Trying to maintain roughly [this ordering](https://github.com/emberjs/ember-inspector/blob/master/ember_debug/libs/render-tree.js#L28-L38) in all places for consistency and make things slightly easier to find).

Also, don't need to check `this.renderNode` as it's guaranteed to be present.

Reply:
@chancancode I would disagree with that ordering, in favor of alphabetizing I think. People are not going to be familiar with what the API returns, that I know of.

Reply:
It's a logical grouping... `isRoot` and `isRouteTemplate` may sound similar but they do pretty different things and doesn't make a lot of sense to sort them next to each other just because of the name happens to match (and then you would have to resort when you refactor and change the name?). A lot of the other files (here, capture-render-tree, render-tree, tests, etc) have this same logical structure/ordering now, roughly from "most important" to "less important" things (subjective, but the ordering is at least consistent everywhere) and I would like to keep it that way. I just liked to that documentation comment because it happens to have enumerated them, but the code structure in other places follows that order (I try to at least).